tests/Core/Tokenizers/Tokenizer/RecurseScopeMapCaseKeywordConditionsTest.inc

```diff
@@ -103,3 +103,17 @@ switch ($value):
     case 1:
         echo 'one';
 endswitch;
+
+// Test for https://github.com/squizlabs/PHP_CodeSniffer/issues/879
```

Comment:
Would it make sense to have a similar test to this one in the `RecurseScopeMapDefaultKeywordConditionsTest` test ?